### PR TITLE
CNV 2.1 release notes

### PR DESCRIPTION
**Pull request by ousleyp**
This PR is intended for the collection of CNV 2.1 release notes. I removed the new and changed features from 2.0 but left the known issues in case we want to move some of them to resolved.

CNV team: please feel free to comment on this PR with requests, including links to the relevant Jiras/BZs that need release notes. Thanks!

Latest preview build (updated existing build, ignore date): http://file.bos.redhat.com/pousley/102119/cnv-2.1-release-notes/cnv/cnv_release_notes/cnv-release-notes.html

### CODE REVIEW

**Comment by jelkosz** [comment]
@ousleyp there is a known issue in the UI: https://bugzilla.redhat.com/show_bug.cgi?id=1753688 (Adding disk via VM Disks tab always adds a disk with 'Filesystem' VolumeMode)

**Comment by dankenigsberg** [comment]
new feature note for VM creation: https://jira.coreos.com/browse/CNV-3061

* When installing CNV, kubemacpool is automatically started.
 * If you define a secondary vNIC and do not specify its MAC address, kubemacpool would automatically allocate a unique address to it.
 * If you are unable to define a secondary vNIC and specify its MAC address so it collides with another vNIC in the cluster

**Comment by aspauldi** [comment]
new note from Fabian for When migration completed "oc get vmi" shows obsolete IP address for pod network
https://bugzilla.redhat.com/show_bug.cgi?id=1686208

**Comment by aspauldi** [comment]
new Known Issue release note from Radim re: kubevirt-storage-defaults configmap is ignored when disk is added on VM Disks tab.
https://bugzilla.redhat.com/show_bug.cgi?id=1755234

**Comment by bgaydosrh** [comment]
Release Notes for Macpool (https://jira.coreos.com/browse/CNV-3061) 
Release Note needed to introduce dashboard and limitations (https://jira.coreos.com/browse/CNV-3240)

Both of these have final drafts of text sent to @ousleyp in email. Petr had asked for another revision today on CNV-3061 today. I made that this morning and am only waiting on an ack from him, but it's in good shape.

**Comment by bgaydosrh** [comment]
Document a workaround for creating a VM for non-admin user using a VM wizard in the UI (BZ1757311: )https://bugzilla.redhat.com/show_bug.cgi?id=1757311.

Sent draft text to @ousleyp.

**Comment by bgaydosrh** [comment]
Doc VM VNC console sometimes does not display content https://bugzilla.redhat.com/show_bug.cgi?id=1757765
Originating bug: https://bugzilla.redhat.com/show_bug.cgi?id=1753606.

Draft text is in https://bugzilla.redhat.com/show_bug.cgi?id=1757765

Waiting for code review from @jelkosz then we should be good to go.

Thanks
Bob

**Comment by alexxa** [comment]
@jelkosz @rhrazdil I think we should mention removing web-ui from 2.1 and including all VM around things in OCP console in these release notes, WDYT?

**Comment by bgaydosrh** [comment]
> @jelkosz @rhrazdil I think we should mention removing web-ui from 2.1 and including all VM around things in OCP console in these release notes, WDYT?

I can add this to [CNV-3240](https://jira.coreos.com/browse/CNV-3240) if needed. For what it's worth, I do think this information should be spelled out explicitly and would welcome collaboration on the wording. Thanks Irina.

**Comment by bgaydosrh** [comment]
404 Not Found instead of a list of operators in UI - https://bugzilla.redhat.com/show_bug.cgi?id=1758207
Originating bug: https://bugzilla.redhat.com/show_bug.cgi?id=1757526
Draft text with markup is in: https://bugzilla.redhat.com/show_bug.cgi?id=1758207

Final text marked for @rthallisey to review in https://bugzilla.redhat.com/show_bug.cgi?id=1758207

Thanks,
Bob

**Comment by aburdenthehand** [comment]
Resources left after cnv uninstall, ns can't be removed, reinstall script will fail - https://bugzilla.redhat.com/show_bug.cgi?id=1757705
Release note + workaround complete

**Comment by ousleyp** [comment]
Issue with acquiring upgrade status: https://bugzilla.redhat.com/show_bug.cgi?id=1759612
Known issue needed; asked Ryan to provide doc text.

**Comment by ousleyp** [comment]
> Resources left after cnv uninstall, ns can't be removed, reinstall script will fail - https://bugzilla.redhat.com/show_bug.cgi?id=1757705
> Release note + workaround complete

KB article to be linked from Known Issue: https://access.redhat.com/articles/4484961

**Comment by ousleyp** [comment]
Known issue, not yet written: [[Bug 1760385](https://bugzilla.redhat.com/show_bug.cgi?id=1760385)] [DOCS] Add to 2.1 release notes: Migration not supported when nodes have different CPU models {BZ 1760028}

**Comment by ousleyp** [comment]
NOTE: Release notes for BZ#1760028 and BZ#1759612 are awaiting review and might need to be edited further.

**Review by alexxa — APPROVED** [review]
Looks good, thanks Pan

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bobfuru) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: s/OpenShift web console/{product-title} web console
We use product title on first instance for web console.

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bobfuru) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: s/ > / -> 
"Use two characters to form the arrow in a series of menu items, ->." (from [Documentation Guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference))

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bobfuru) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: s/card of the dashboard/dashboard card

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bobfuru) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: s/there is a chance/it is possible 
or something similar, "chance" sounds odd for computing :)

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bobfuru) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: I haven't seen a `+` splitting object examples like this in our docs, but maybe we have. Should this be split into two code blocks?

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bobfuru) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: I get an `access denied` from this link.

**Review by bobfuru — COMMENTED** [review]
A few minor items, including a link that doesn't work for me.

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (ousleyp) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Good catch! But I believe we're waiting to publish this article until we push the 2.1 docs. Is that correct, @aburdenthehand?

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (ousleyp) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Good catch, I do think that was supposed to be two code blocks.

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (alexxa) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: It works for me. One is supposed to log in to access that page =)

Comment (bobfuru) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Oof, thought I was already logged in. Works for me, too!

**Comment by bobfuru** [comment]
LGTM!

**Comment by bergerhoffer** [comment]
After chatting with @ousleyp, this should be carried forward to 4.3 as well.

**Pull request merged**
